target/linux/ramips/dts/mt7628an_iptime.dtsi

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	aliases {
```

Comment:
Since CPU led isn't defined in dtsi I'd suggest moving this aliases node into device dts as well.

Reply:
Done.